docs/sources/evpn.md

```diff
@@ -18,6 +18,7 @@ still very experimental.
 - [YABGP](#yabgp)
   - [Configuration](#configuration-1)
   - [Advertising EVPN route](#advertising-evpn-route-1)
+- [BGP Type 2 and Type 5 Routes](#bgp-type-2-and-type-5-routes)
```

Comment:
Thank you for great PR!

In this section, you describe about Router's Mac Option, so changing the title makes it more intuitive, "Router's Mac Option", etc.
And I think It's better to change the position of this section.

* CLI Syntax
  * Ethernet Segment Identifier
  * Ethernet Auto-discovery Route
  * MAC/IP Advertisement Route
  * Inclusive Multicast Ethernet Tag Route
  * Ethernet Segment Route
  * IP Prefix Route
* Peference
  * Router's Mac Option
* BaGPipe
  * Configuration
  * Advertising EVPN route
(...snip...)